cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/hdfs/HdfsStorageClient.java

```diff
@@ -48,4 +49,20 @@ public FileSystem getNativeClient() {
   public StorageType.Type getStorageType() {
     return HDFS_TYPE;
   }
+
+  /**
+   * Checks if the path exists on the backend storage. Scheme is not prefix for local and hdfs
```

Comment:
nit: I am having a hard time parsing the language "scheme is not prefix". Would you please consider paraphrasing to something like "Scheme is not specified in the path for local and hdfs storage".